### PR TITLE
Feature/broadcast chat

### PR DESCRIPTION
**Pull request by xkevin190**
the public chat channel was added again

### CODE REVIEW

File: __tests__/Views/configuration/AdministrativeComponent-test.js [hunk 1]
```diff
@@ -144,4 +147,25 @@ describe('test Administrative component', () => {
...
+      broadcast: false,
+      enableBroadcast: mockEnableBroadcast
+    });
+    wrapper.find('Styled(Switch)').at(2).props().onTouchEnd();
+
+    expect(mockEnableBroadcast.mock.calls.length).toBe(1);
+  });
+
+
+  test('disabled bradcast chat', () => {
```
Comment (luisan00) [inline __tests__/Views/configuration/AdministrativeComponent-test.js:162]:
```suggestion
  test('disabled broadcast chat', () => {
```

**Review by luisan00 — CHANGES_REQUESTED** [review]
just a little change. without importance

File: __tests__/Views/configuration/AdministrativeComponent-test.js (hunk 1 above)
Comment (xkevin190) [reply __tests__/Views/configuration/AdministrativeComponent-test.js:162]: ok

**Review by jeandudey — APPROVED** [review]

**Review by luisan00 — APPROVED** [review]
epale!

**Pull request merged**